deploy/helm/sumologic/templates/events/otelcol/statefulset.yaml

```diff
@@ -100,6 +100,10 @@ spec:
       containers:
       - name: otelcol
         image: {{ .Values.otelevents.image.repository }}:{{ .Values.otelevents.image.tag }}
+{{- if .Values.sumologic.pullSecrets }}
+        imagePullSecrets:
```

Comment:
Doesn't `otelevents` already use the `sumologic.pullSecrets` by using the common service account?